ui.content/src/main/content/jcr_root/var/.content-cloud.xml

```diff
@@ -0,0 +1,5 @@
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
```

Comment:
You should be able to remove this if you add the `exclude` rule.

Reply:
+1 this should be a reference, lets only add things which are really needed.